cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java

```diff
@@ -50,7 +50,7 @@ public Cocos2dxWebView(Context context, int viewTag) {
         this.mJSScheme = "";
 
         this.setFocusable(true);
-        this.setFocusableInTouchMode(true);
+        this.setFocusableInTouchMode(false);
```

Comment:
The issue you described is existed, and your changes fix the issue, but it has a bad effect on others. 
when you set `this.setFocusableInTouchMode(false);`, some item in WebView can't get focus. For example load url `https://www.baidu.com`, the search bar can't  get focus when you touch on it.